Libraries/Components/ScrollView/ScrollViewStickyHeader.js

```diff
@@ -181,7 +181,7 @@ class ScrollViewStickyHeader extends React.Component<Props, State> {
     // Fabric Detection
     // eslint-disable-next-line dot-notation
     const isFabric = !!(
-      this._ref && this._ref['_internalInstanceHandle']?.stateNode?.canonical
```

Comment:
We'll have to revert this. As I've learned from @TheSavior our babel transform has issues with `_` prefixed properties